test/test_helper.rb

```diff
@@ -1,7 +1,13 @@
 ENV['RAILS_ENV'] ||= 'test'
+
 require File.expand_path('../../config/environment', __FILE__)
+require 'codeclimate-test-reporter'
+require 'pullreview/coverage_reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.